index.html

```diff
@@ -57,6 +57,260 @@ <h2>I'm a Software Developer</h2>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="projects">
+          <div class="project-details">
+            <div class="project-title">
+              <h4>Profesional Art Printing Data</h4>
+            </div>
+            <div class="support-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <div class="pro-language">
+              <ul>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="projects">
+          <div class="project-details">
+            <div class="project-title">
+              <h4>Profesional Art Printing Data</h4>
+            </div>
+            <div class="support-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. Has been the industry's standard.
+              </p>
+            </div>
+            <div class="pro-language">
+              <ul>
+                <li><a href="#">HTML</a></li>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="pro-btn">
+            <span><a href="#">See project</a></span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div>
+        <div class="about">
+          <div class="about-text">
+            <h4>About me</h4>
+          </div>
+          <div class="about-description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="about-footer">
+            <div class="about-btn">
+              <span><a href="#">Get my resume</a></span>
+            </div>
+            <div class="about-border"></div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <ul class="language-sec">
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec.svg" alt="rec" />
```

Comment:
- [ ] Your layout is currently broken, please try to fix it before, so that your design matches the Figma design

![layout](https://user-images.githubusercontent.com/102636489/227739356-e4d1ecbf-e121-43b1-aa25-5ac740a2208a.png)

Reply:
Done :heavy_check_mark: 

![Screen Shot 2023-03-26 at 00 20 23](https://user-images.githubusercontent.com/13707896/227742572-5c5401cd-e548-4ba6-ba7e-1ec8b27fa91c.png)